### PR TITLE
Little `IndividualFriendlyFire` fix

### PR DESCRIPTION
**Pull request by IRacle1**

### CODE REVIEW

File: Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs [hunk 1]
```diff
@@ -291,10 +279,7 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                 // targetReferenceHub
                 new(OpCodes.Ldloc_3),
 
-                new(OpCodes.Ldarg_0),
```
Comment (Undid-Iridium) [inline Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs:294]: This patch is disabled - why not just delete at this point? If it's needed again it's in the commit history

File: Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs [hunk 2]
```diff
@@ -123,7 +111,7 @@ public static bool CheckFriendlyFirePlayerRules(ReferenceHub attackerHub, Refere
                         }
                     }
                 }
-                else if(!attacker.UniqueRole.Equals(string.Empty))
+                else if(!string.IsNullOrEmpty(attacker.UniqueRole))
```
Comment (Undid-Iridium) [inline Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs:114]: This should never be null unless a developer screwed up but sure - this works as well

File: Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs (hunk 1 above)
Comment (IRacle1) [reply Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs:294]: Yea, you right

File: Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs (hunk 2 above)
Comment (IRacle1) [reply Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs:114]: You're right, but `IsNullOrEmpty` makes more sense here then `Equals`, and an extra check for `null` wouldn't hurt

File: Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs (hunk 1 above)
Comment (IRacle1) [reply Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs:294]: Although no, it is marked as TODO, so let it exist(maybe ask dev bois later)

Comment (Undid-Iridium) [reply Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs:294]: > Although no, it is marked as TODO, so let it exist(maybe ask dev bois about removing it later)

That was Joker, and it was left there by me incase I screwed up the ff stuff for grenades, thus far I have not heard a report so, I am going to assume it's fine to remove. Though you can double check

Comment (IRacle1) [reply Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs:294]: Sure, you right

**Review by BuildBoy12 — APPROVED** [review]

**Comment by IRacle1** [comment]
That wasn't bug, but okay

**Comment by NaoUnderscore** [comment]
That was a fix, that's all.

**Pull request merged**